core/src/main/resources/lib/hudson/queue.jelly

```diff
@@ -103,7 +103,8 @@ THE SOFTWARE.
   <!-- schedule updates only for the full page reload -->
   <j:if test="${ajax==null and h.hasPermission(app.READ)}">
     <script defer="defer">
-      refreshPart('buildQueue',"${rootURL}/${h.hasView(it,'ajaxBuildQueue')?it.url:''}ajaxBuildQueue");
+      refreshPart('buildQueue',"${rootURL}/${h.hasView(it,'ajaxBuildQueue')?it.url:''}ajaxBuildQueue",
+      ${h.ajaxRefreshInterval});
```

Comment:
@Wadeck Is this reasonably transferable to separate JS like you've been working on?

Reply:
Of course, but not in this PR :) My "spider sense" always hurts when I see new injection of variable like this ;)